src/plugins/data_source_management/public/components/drop_down_header.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiTitle, EuiFlexGroup, EuiFlexItem, EuiLink } from '@elastic/eui';
+import React from 'react';
+import { ApplicationStart } from 'opensearch-dashboards/public';
+import { DSM_APP_ID } from '../plugin';
+
+interface DataSourceOptionItemProps {
+  dataSourceCount: number;
+  application?: ApplicationStart;
+}
+
+export const DataSourceDropDownHeader: React.FC<DataSourceOptionItemProps> = ({
+  dataSourceCount,
```

Comment:
Since other components will also use this drop down header, can we instead have `totalDataSourceCount` and `activeDataSourceCount?`? This added option is for displaying the number of selected options/total options:

```Typescript
`DATA SOURCE (1)`
`DATA SOURCES (${totalDataSourceCount})`

`DATA SOURCES (${activeDataSourceCount}/${totalDataSourceCount})`
```

Reply:
Yes, I updated the logic to include this, and updated the test cases as well